WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeLearnHowView.swift

```diff
@@ -0,0 +1,242 @@
+import SwiftUI
+
+/// View to display the learn how sheet of Blaze campaign creation
+///
+struct BlazeLearnHowView: View {
+    @Binding var isPresented: Bool
+
+    private let steps: [Step] = [.init(number: Localization.ChooseProduct.number,
+                                       title: Localization.ChooseProduct.title,
+                                       subtile: Localization.ChooseProduct.subtitle),
+                                 .init(number: Localization.CustomizeTargeting.number,
+                                       title: Localization.CustomizeTargeting.title,
+                                       subtile: Localization.CustomizeTargeting.subtitle),
+                                 .init(number: Localization.SetYourBudget.number,
+                                       title: Localization.SetYourBudget.title,
+                                       subtile: Localization.SetYourBudget.subtitle),
+                                 .init(number: Localization.QuickReview.number,
+                                       title: Localization.QuickReview.title,
+                                       subtile: Localization.QuickReview.subtitle),
+                                 .init(number: Localization.GoLive.number,
+                                       title: Localization.GoLive.title,
+                                       subtile: Localization.GoLive.subtitle)]
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            titleBlock
+
+            stepBlock
+        }
+    }
+}
+
+private extension BlazeLearnHowView {
+    var titleBlock: some View {
+        ZStack(alignment: .center) {
```

Comment:
I don't think we need to use `ZStack` in this case. How about a simple `HStack` for both the button and the title?

```Swift
HStack {
    Button(action: {}, label: { ... })
    Spacer()
    Text(String(format: Localization.title, "Blaze"))
         .fontWeight(.semibold)
          .headlineStyle()
    Spacer()
}
.padding(.horizontal, Layout.TitleBlock.closeButtonHorizontalPadding)
.padding(.top, Layout.TitleBlock.topPadding)

```

Reply:
I used a `ZStack` because this setup allows us to align the title text at the center. If we use a `HStack`, the title text won't be perfectly centered. What do you think?

Reply:
You're right, I tried the solution and found that the title is not perfectly centered. What I don't like about ZStack or `overlay` is that the views can overlap each other (e.g when the text is long/large enough):
<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/f6715f6e-d380-48cc-95f7-81c3edf04ac4" width=320 />

I was thinking maybe we could use a navigation bar and [hide the background](https://developer.apple.com/documentation/swiftui/view/toolbarbackground(_:for:)-7lv0f), but the API is not available until iOS 16. 

I don't have other suggestions for this.

Reply:
Good point about the overlapping. I replaced the `ZStack` with `HStack` as per your suggestion. https://github.com/woocommerce/woocommerce-ios/pull/11583/commits/80b8b8fa51bb91430615e745418c7dd0318ba75e